cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -395,6 +395,16 @@ install() {
         say "  $_bin_name"
     done
 
+    {% for source, dests in aliases | items -%}
+    local _source="$_install_dir/{{ source }}"
+    {% for dest in dests -%}
+    local _dest="$_install_dir/{{ dest }}"
+    if [ -f "$_source" ]; then
+        ln -s "$_source" "$_dest"
+    fi
```

Comment:
Yeah, I mostly punted on this because `sh` doesn't have hash tables so the mapping logic gets hairier. Now that I think of it a function that returns aliases per binary shouldn't be hard though.

Reply:
There's a bug with the implementation right now, where the target triples we put in the executable zip fragment don't make it into the set being followed by the aliases list right now. I'll fix this tomorrow.